From 62172e95a301f98eefd0a508ca3678a6d18c1916 Mon Sep 17 00:00:00 2001
From: James Kosin <jkosin@apache.org>
Date: Wed, 9 Nov 2011 02:11:26 +0000
Subject: [PATCH] OPENNLP-361: Added quotes around setting of JAVA_CMD for
 possible spaces ... Thanks to Aliaksandr Autayeu for providing the patch.

git-svn-id: https://svn.apache.org/repos/asf/incubator/opennlp/trunk@1199598 13f79535-47bb-0310-9956-ffa450edef68
---
 opennlp-distr/src/main/bin/opennlp.bat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opennlp-distr/src/main/bin/opennlp.bat b/opennlp-distr/src/main/bin/opennlp.bat
index cb7fa2e3b..cdaad0448 100644
--- a/opennlp-distr/src/main/bin/opennlp.bat
+++ b/opennlp-distr/src/main/bin/opennlp.bat
@@ -25,7 +25,7 @@ IF "%JAVA_CMD%" == "" (
 	IF "%JAVA_HOME%" == "" (
 		SET JAVA_CMD=java 
 	) ELSE (
-		SET JAVA_CMD=%JAVA_HOME%\bin\java
+		SET JAVA_CMD="%JAVA_HOME%\bin\java"
 	)
 )
 
